Packet++/src/SSLHandshake.cpp

```diff
@@ -1228,6 +1228,19 @@ SSLHandshakeMessage* SSLHandshakeMessage::createHandshakeMessage(uint8_t* data,
 		return nullptr;
 
 	ssl_tls_handshake_layer* hsMsgHeader = (ssl_tls_handshake_layer*)data;
```

Comment:
it'd be great to add a test with your pcap file. 

We should also fix the test you mentioned in the PR comment: https://github.com/seladb/PcapPlusPlus/pull/1087#issuecomment-1452378651

Reply:
I added a test with my pcap file of #1085 in the commit [fa1fb91](https://github.com/seladb/PcapPlusPlus/pull/1087/commits/fa1fb9149b61b20fd545fb59413e25ae8d009dc9)

The other test should be fixed now as well.